config/metadata_report_config.go

```diff
@@ -103,7 +103,7 @@ func startMetadataReport(metadataType string, metadataReportConfig *MetadataRepo
 	if url, err := metadataReportConfig.ToUrl(); err == nil {
 		instance.GetMetadataReportInstance(url)
 	} else {
-		return perrors.New("MetadataConfig is invalid!")
+		return perrors.Wrap(err, "Start MetadataReport failed.")
```

Comment:
why use Wrap instead of New?

Reply:
fuction metadataReportConfig.ToUrl() returns the err, but original codes ignore it and return a new error, actual err is the same as new error. so wrap it to get more infomation.

Reply:
> why use Wrap instead of New?

fuction metadataReportConfig.ToUrl() returns the err, but original codes ignore it and return a new error, actual err is the same as new error. so wrap it to get more infomation.